spec/requests/assisted_digital_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 require 'gds_api/test_helpers/support_api'
-
+require 'webmock'
```

Comment:
Webmock is already included on tests of `type: :request` so you can drop this and the `Webmock.` from `stub_request` as it's exposed as just `stub_request`